esmvaltool/preprocessor/_regrid.py

```diff
@@ -197,6 +197,17 @@ def regrid(cube, target_grid, scheme):
                 [coord] = coords
                 cube.remove_coord(coord)
 
+    # Guess bounds for area weighted regridding if necessary
```

Comment:
@jvegasbsc Can you comment on this? I though cmorized data always contains bounds?